### PR TITLE
Allow Money.zero to specify a currency for strictness

### PR DESCRIPTION
**Pull request by elsom25**
Sometimes we want a specific currency around our zero money instead of a null currency

### CODE REVIEW

File: lib/money/money.rb [hunk 1]
```diff
@@ -23,8 +23,9 @@ def new(value = 0, currency = nil)
     end
     alias_method :from_amount, :new
 
-    def zero
-      new(0, NULL_CURRENCY)
+    def zero(currency = nil)
+      currency = Helpers.value_to_currency(currency) if currency.present?
+      new(0, currency.presence || NULL_CURRENCY)
```
Comment (grcooper) [inline lib/money/money.rb]: why `currency.presence` and not iso? I'm just not well versed in what a currency is and couldn't find presence in the currency model

Comment (elsom25) [reply lib/money/money.rb]: [`#presence`](http://guides.rubyonrails.org/active_support_core_extensions.html#presence) is from ActiveSupport and basically coerces empty values to nil. We could probably do the simple `currency || NULL_CURRENCY`, but I like using presence to protect against weird edges (i.e. `.zero("")`

Comment (elfassy) [reply lib/money/money.rb]: this should do it:
```ruby
def zero(currency = NULL_CURRENCY)
  new(0, currency)
end
```

Comment (elsom25) [reply lib/money/money.rb]: 🤦‍♂️

**Pull request merged**